include/curl/typecheck-gcc.h

```diff
@@ -362,7 +365,7 @@ _CURL_WARNING(_curl_easy_getinfo_err_curl_off_t,
    (option) == CURLOPT_SSL_CTX_DATA ||                                        \
    (option) == CURLOPT_WRITEDATA ||                                           \
    (option) == CURLOPT_RESOLVER_START_DATA ||                                 \
-   (option) == CURLOPT_CURLU ||                                               \
```

Comment:
What's with this option now? Did you remove it on purpose?

Reply:
I hesitated on how to deal with it. The macro there is specifically said to be for pointer arguments to callbacks so `CURLOPT_CURLU` was wrongly put there to begin with, even if it also takes a pointer. This rights that wrong. Not sure it makes any particular difference for anyone.